### PR TITLE
config(ticdc): make table actor batch size configurable

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/3881

this is a part of table pipeline actor model refactor https://github.com/pingcap/tiflow/issues/3881

### Check List 

Tests 
add a changefeed level configuration to set the table actor batch size

 - Unit test
 - Integration test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
Please refine your PR title.

File: pkg/config/actor.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+// ActorConfig represents config used for table actor
+type ActorConfig struct {
```
Comment (overvenus) [inline pkg/config/actor.go:17]: What about `TableActorConfig`?

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: pkg/config/replica_config.go [hunk 2]
```diff
@@ -49,6 +49,9 @@ var defaultReplicaConfig = &ReplicaConfig{
 		FlushIntervalInMs: 1000,
 		Storage:           "",
 	},
+	TableActor: &TableActorConfig{
```
Comment (overvenus) [inline pkg/config/replica_config.go:52]: Could you add to Debug section for now? We can move it out when table actor is GA.

**Review by overvenus — APPROVED** [review]
Approve in advance, see my comments.

File: pkg/config/replica_config.go (hunk 2 above)
Comment (sdojjy) [reply pkg/config/replica_config.go:52]: ok

File: pkg/config/actor.go (hunk 1 above)
Comment (sdojjy) [reply pkg/config/actor.go]: done

File: cdc/processor/pipeline/actor_node_context.go [hunk 3]
```diff
@@ -20,30 +20,31 @@ import (
...
 	"github.com/pingcap/tiflow/pkg/actor/message"
+	"github.com/pingcap/tiflow/pkg/config"
 	"github.com/pingcap/tiflow/pkg/context"
 	"github.com/pingcap/tiflow/pkg/pipeline"
 	"go.uber.org/zap"
 )
 
-// send a tick message to actor if we get 32 pipeline messages
-const messagesPerTick = 32
+// send a tick message to actor if we get 32 pipeline events
```
Comment (hi-rustin) [inline cdc/processor/pipeline/actor_node_context.go:29]: It is recommended to start a go comment with a variable name or a function name.

**Review by hi-rustin — APPROVED** [review]
rest LGTM

File: cdc/processor/pipeline/actor_node_context.go (hunk 3 above)
Comment (sdojjy) [reply cdc/processor/pipeline/actor_node_context.go]: done

File: cdc/processor/pipeline/actor_node_context.go [hunk 4]
```diff
@@ -20,30 +20,31 @@ import (
...
 	"github.com/pingcap/tiflow/pkg/actor/message"
+	"github.com/pingcap/tiflow/pkg/config"
 	"github.com/pingcap/tiflow/pkg/context"
 	"github.com/pingcap/tiflow/pkg/pipeline"
 	"go.uber.org/zap"
 )
 
-// send a tick message to actor if we get 32 pipeline messages
-const messagesPerTick = 32
+// defaultEventBatchSize send a tick message to actor if we get 32 pipeline events
```
Comment (hi-rustin) [inline cdc/processor/pipeline/actor_node_context.go:29]:
```suggestion
// defaultEventBatchSize specifies that if we get 32 pipeline events, a tick message is sent to the actor.
```

**Pull request merged**